src/stores/FilterBarStore.js

```diff
@@ -190,10 +192,31 @@ export class FilterBarStore {
     })
   }
 
-  updateFilter(filterUid, propKey, propValue) {
-    this.filters[filterUid][propKey] = propValue;
-    if(propKey === 'value')
-      this.deactivateQuickFiltersBasedOnFilterValue(filterUid, propValue, this.activeQuickFilters());
+  disableFilter(groupKey, inputKey) {
+    this.activeFilters[groupKey].splice(inputKey, 1);
+    if (this.activeFilters[groupKey].length === 0) {
+      this.activeFilters.splice(groupKey, 1);
+    }
+
+    this.emitChange();
+  }
+
+  updateFilter(groupKey, inputKey, value) {
+    //this.deactivateQuickFiltersBasedOnFilterValue(filterUid, propValue, this.activeQuickFilters());
+
+    this.activeFilters[groupKey][inputKey].value = value;
+  }
+
+  addGroupFilter(groupKey, filterUid) {
```

Comment:
As mentioned elsewhere, changing the order of these params would allow you to pass `groupKey` optionally, then just check for `undefined`.

Reply:
Yep, that's a good point! I will follow your suggestion 👌